charts/appdaemon/ci/ct-values.yaml

```diff
@@ -0,0 +1,8 @@
+env:
+  TZ: "Africa/Cairo"
+  LATITUDE: "29.9792"
+  LONGITUDE: "31.1342"
+  ELEVATION: "64"
+  DASH_URL: "http://0.0.0.0:5050"
+ingress:
```

Comment:
This shouldn't be necessary as it's the default, right?

Reply:
appdaemon needs all those environment variables set or it will not pass CI :(

Reply:
I meant the ingress, maybe I selected the wrong line 😊

Reply:
Got it, I updated to enable ingress